tsconfig.json

```diff
@@ -31,8 +31,8 @@
     "**/*.test.ts",
     "**/*.test.tsx",
     "**/*.stories.tsx",
-    "./src/setupTests.ts",
-    "./stories/**"
+    "src/**",
+    "src/commons/stories/**"
   ],
-  "files": ["./src/index.ts"]
-}
\ No newline at end of file
+  "files": ["src/components/index.ts"]
```

Comment:
This configuration causes the build to be missing all the component barrels. A better config would be to remove `files` and have  the `include` option be `src/components/**/index.tsx`

Reply:
Nice catch!